node-gui/src/backend/backend_impl.rs

```diff
@@ -376,7 +376,8 @@ impl Backend {
             .get_mut(&wallet_id)
             .ok_or(BackendError::UnknownWalletIndex(wallet_id))?
             .controller
-            .synced_controller(account_index)
+            // TODO: add option to select from GUI
+            .synced_controller(account_index, ControllerConfig { in_top_x_mb: 5 })
```

Comment:
Please, no magic numbers in the middle of the code.

Reply:
added the constant at the top with a comment